src/main/java/com/programmers/ticketparis/controller/member/CustomerController.java

```diff
@@ -0,0 +1,39 @@
+package com.programmers.ticketparis.controller.member;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.programmers.ticketparis.dto.ApiResponse;
+import com.programmers.ticketparis.dto.member.CustomerCreateRequest;
+import com.programmers.ticketparis.dto.member.CustomerResponse;
+import com.programmers.ticketparis.service.member.CustomerService;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@RequestMapping("/api/customers")
+@RequiredArgsConstructor
+@Slf4j
+public class CustomerController {
+    private final CustomerService customerService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public void createAccount(@Valid @RequestBody CustomerCreateRequest customerCreateRequest) {
+        customerService.createAccount(customerCreateRequest);
+    }
+
+    @GetMapping("/{customerId}")
+    public ApiResponse<CustomerResponse> findById(@PathVariable("customerId") Long customerId, HttpServletRequest httpServletRequest) {
```

Comment:
`p1`

```suggestion
    public ApiResponse<CustomerResponse> findCustomerById(@PathVariable("customerId") Long customerId, HttpServletRequest httpServletRequest) {
```
팀 컨벤션으로 네이밍 변경부탁드립니다! 🙏

Reply:
아 어제 결정하고는 깜박했네요. 감사드립니다~
f263d402ff4f18c61fa21498a1b9ff8505e54b69